backend/app/adapter/gqlapi/resolver/mutation.go

```diff
@@ -35,21 +36,23 @@ func (m Mutation) AuthMutation(args *AuthMutationArgs) (*AuthMutation, error) {
 		return nil, ErrNotHuman{}
 	}
 
-	authMutation := newAuthMutation(args.AuthToken, m.authenticator, m.changeLog, m.urlCreator)
+	authMutation := newAuthMutation(args.AuthToken, m.authenticator, m.changeLog, m.urlCreator, m.urlUpdater)
```

Comment:
For your information, we will inject `newAuthMutation` directly in the future. So that won't be logger, urlCreator, etc. on Mutation.

Reply:
Sounds good I will update it